agent/config/pbench-agent.cfg.example

```diff
@@ -7,4 +7,4 @@ pbench_web_server = pbench.example.com
 
 [config]
 path = %(pbench_install_dir)s/config
-files = pbench-agent-default.cfg
+files = pbench-agent.cfg
```

Comment:
This should be pbench-agent-default.cfg, as it was before.

Reply:
Looks like this change was taken care of with the latest commit.